api/tests/test_assets_assignee_api.py

```diff
@@ -67,8 +67,6 @@ def test_can_get_assets_assignee(self, mock_verify_token):
         response = client.get(
             self.asset_assignee_url,
             HTTP_AUTHORIZATION="Token {}".format(self.token_user))
-        self.assertEqual(len(response.data['results']),
-                         AssetAssignee.objects.count())
```

Comment:
This is because once a workplace or a department is created, they are both registered as `AssetAssignee` but AssetAssignee has no location field it depends on either a `User` or a `Workspace` location.
Since  a Department has no location, I guess we will return all departments right??
So I had removed the test since `AssetAssignee.objects.count()` would nevel equal `len(response.data['results'])`. I have fixed this however.